aeron-driver/src/main/java/io/aeron/driver/Main.java

```diff
@@ -0,0 +1,121 @@
+package io.aeron.driver;
+
+import io.aeron.Aeron;
+import io.aeron.Publication;
+import io.aeron.Subscription;
+import io.aeron.logbuffer.FragmentHandler;
+import org.agrona.concurrent.SleepingIdleStrategy;
+import org.agrona.concurrent.UnsafeBuffer;
+
+import java.nio.ByteBuffer;
+
+public class Main
```

Comment:
Is this a debug code? If not, then it should be turned into a a test instead.

Reply:
It will also be removed.